client/verta/verta/_registry/model.py

```diff
@@ -122,6 +122,15 @@ def _create_proto_internal(cls, conn, ctx, name, desc=None, tags=None, attrs=Non
         return registered_model
 
     def add_labels(self, labels):
+        """
+        Adds multiple labels to this Registered Model.
+
+        Parameters
+        ----------
+        labels : list of str
+            labels to add.
```

Comment:
Capitalize the descriptions of `Parameters`/`Returns`.
```suggestion
            Labels to add.
```